WordPress/wordcamp.org
# Docs: Add the Termination clause to the Sponsorship Agreement

Fixes #827.

### Screenshots

The new section is added to the doc as `#7`

<img width="816" alt="Screenshot 2023-03-28 at 11 40 59 AM" src="https://user-images.githubusercontent.com/541093/228292348-0756aa54-f765-44d6-8f86-229e2d0d4721.png">

### How to test the changes in this Pull Request:

1. Go to Docs, select Sponsorship Agreement
2. Fill out the fields & generate the doc
3. It should have a Termination section
